src/PublicApiGeneratorTests/Struct_readonly.cs

```diff
@@ -12,6 +12,19 @@ public void Should_output()
 @"namespace PublicApiGeneratorTests.Examples
 {
     public readonly struct ReadonlyStruct { }
+}");
+        }
+
+        [Fact]
+        public void Should_output_with_ctor()
```

Comment:
Could you also add a similar test for a class with a static constructor?

Reply:
done